speedcord/values.py

```diff
@@ -4,4 +4,4 @@
 Don't touch version, it is automatically generated.
 """
 
-version = "0.1.57"
\ No newline at end of file
+version = "0.1.57"
```

Comment:
This should not be included in this pr. The pr this was included in was rejected